spec/controllers/devise/saml_sessions_controller_spec.rb

```diff
@@ -75,6 +129,20 @@ def require_no_authentication
       expect(response).to redirect_to '/users/saml/sign_in'
     end
 
+    context "with a specified idp" do
```

Comment:
In this new context, `idp_sign_ou` redirects to the SSO target url of the IdP that corresponds to the `Issuer` in the parsed SAML logout request when `Devise.idp_settings_adapter` is set.